core/dbt/events/types.py

```diff
@@ -994,6 +997,649 @@ def cli_msg(self) -> str:
         return "Done."
 
 
+@dataclass
+class ServingDocsPort(InfoLevel, CliEventABC):
+    port: str
+
+    def cli_msg(self) -> str:
+        return f"Serving docs at 0.0.0.0:{self.port}"
```

Comment:
is there a place this address is used in the code? can we pass it in instead of having to maintain that the message is in sync?

Reply:
Not in the form of a variable but I can use one and sync it up.

Reply:
I think that would be a great improvement that's worth doing now since it doesn't change the message output.